app/uk/gov/hmrc/helptosavefrontend/views/register/create_account.scala.html

```diff
@@ -14,55 +14,83 @@
  * limitations under the License.
  *@
 
-@import uk.gov.hmrc.helptosavefrontend.controllers.routes
+@import cats.syntax.eq._
+@import uk.gov.hmrc.helptosavefrontend.forms.BankDetails
+@import uk.gov.hmrc.helptosavefrontend.views.html.helpers
 @import uk.gov.hmrc.helptosavefrontend.models.HtsContext
-@import uk.gov.hmrc.helptosavefrontend.views.html.helpers.form
-@import uk.gov.hmrc.helptosavefrontend.views.html.helpers.ga_submit_button
-@import uk.gov.hmrc.helptosavefrontend.views.html.helpers._
-@import uk.gov.hmrc.helptosavefrontend.models.eligibility.EligibilityReason
-@import uk.gov.hmrc.helptosavefrontend.models.eligibility.EligibilityReason._
 @import uk.gov.hmrc.helptosavefrontend.config.FrontendAppConfig
+@import uk.gov.hmrc.helptosavefrontend.views.helpers.errorPrefix
+@import uk.gov.hmrc.helptosavefrontend.views.html.helpers._
+@import uk.gov.hmrc.helptosavefrontend.controllers.routes
+@import uk.gov.hmrc.helptosavefrontend.models.HTSSession._
+@import cats.syntax.show._
+@import uk.gov.hmrc.helptosavefrontend.models.userinfo.UserInfo.localDateShow
 
-@(eligibilityReason: EligibilityReason)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
+@(eligibleWithUserInfo: EligibleWithUserInfo, email: String, bankDetails: BankDetails)(implicit htsContext: HtsContext, request: Request[_], messages: Messages, appConfig: FrontendAppConfig)
 
-@title = @{
-  Messages("hts.register.create_account.title.h1")
-}
+@title = @{Messages("hts.cya.title")}
 
-@conditionsFromReason = {
-  @eligibilityReason match {
-    case UCClaimantAndIncomeSufficient | EntitledToWTC(true) => {
-        <li>@{Messages("hts.register.create_account.condition-uc")}</li>
-    }
-    case EntitledToWTC(false) => {
-        <li>@{Messages("hts.register.create_account.condition-wtc")}</li>
-    }
-  }
+@display2CharFormat(str: String) = @{
+  str.filterNot((x: Char) => x.isWhitespace || x.toString.matches("-")).grouped(2).mkString(" ")
 }
 
 @uk.gov.hmrc.helptosavefrontend.views.html.main_template(
-  title = Messages("hts.global.title", title, "", Messages("hts.global.title-suffix")),
-  bodyClasses = None) {
+title = Messages("hts.global.title", title, "", Messages("hts.global.title-suffix")),
+sidebarLinks = None,
+bodyClasses = None) {
+
+@navigate_back(Some(routes.BankAccountController.getBankDetailsPage().url))
 
-  @navigate_back(Some(routes.RegisterController.checkDetails().url))
+  <h1>@title</h1>
 
-  <h1 class="heading-large">
-    @title
-  </h1>
+  <p class="lede">@Html(Messages("hts.cya.intro"))</p>
 
-  <p class="lede">
-    @Messages("hts.register.create_account.p1")
-  <p>
+  @summary_section_head(heading = Messages("hts.cya.your-details"))
```

Comment:
is `cya` check your answers? They're not really checking their answers if it is...

Reply:
It is, the form pattern is referred to as check you answers, but I can update this.